domain/src/main/java/co/nimblehq/compose/crypto/domain/model/TrendingItem.kt

```diff
@@ -0,0 +1,38 @@
+package co.nimblehq.compose.crypto.domain.model
+
+data class TrendingItem(
```

Comment:
Same, please consider merge it to `CoinItem`, we can keep some extra trending coin fields as nullable 🤔